### PR TITLE
Fix zerochats

### PR DESCRIPTION
**Pull request by suprko**
### PR Checklist

- [x] I have merged `dev` into my branch before submitting
- [x] I have checked with either the repo owner or person who previously worked on this feature to ensure correctness
- [x] I have ensured that I am not using any English strings, only `t('translation_keys)`
- [x] The pull request name follows the standard commit guidelines, e.g. `feat(files): filter by file type`

### Relevant user stories or bugs
Zero-chat text localized, styles cleaned up. Should not affect anything other than Chat view when no rooms/chats exist yet

### CODE REVIEW

File: src/ui/chat/components/NoChatSelected.jsx [hunk 1]
```diff
@@ -28,14 +29,14 @@ class NoChatSelected extends React.Component {
...
         return (
             <div className="zero-message new-dm create-new-chat">
-                <div style={{
-                    marginTop: '168px',
-                    maxWidth: '600px'
-                }} >
-                    <h2 className="display-2">No chats? <strong>No problem!</strong></h2>
-                    <div className="title">Enter a contacts email or user name to get started</div>
+                <div className="zero-message-content">
+                    <T k="title_zeroChat" className="zero-chat-header" tag="div">{textParser}</T>
```
Comment (anri-asaturov) [inline src/ui/chat/components/NoChatSelected.jsx]: We have <b> tag for this, just add it to locale file string.

File: src/ui/chat/components/NoChatSelected.scss [hunk 2]
```diff
@@ -18,4 +15,37 @@
...
+    .chat-creation-header-container {
+      border: none;
+    }
+
+    .email-invite-container {
+      padding-right: 0;
+    }
+  }
+
+  .usernotfound-container {
```
Comment (anri-asaturov) [inline src/ui/chat/components/NoChatSelected.scss]: this class name is not kebab-cased properly :-D

File: src/ui/chat/components/NoChatSelected.jsx (hunk 1 above)
Comment (suprko) [reply src/ui/chat/components/NoChatSelected.jsx]: I just add it directly to `/app/package.json`?

Comment (anri-asaturov) [reply src/ui/chat/components/NoChatSelected.jsx]: https://github.com/PeerioTechnologies/peerio-copy/blob/icebear/icebear_en.json#L66

**Comment by suprko** [comment]
@anri-asaturov changes done

**Pull request merged**